### PR TITLE
Update definitions

### PR DESCRIPTION
**Pull request by lukaspie**

### CODE REVIEW

File: .github/workflows/plugin_test.yaml [hunk 1]
```diff
@@ -28,13 +28,13 @@ jobs:
             branch: main
             tests_to_run: tests/.
           - plugin: pynxtools-mpes
-            branch: main
+            branch: update-to-new-nxmpes
```
Comment (domna) [inline .github/workflows/plugin_test.yaml:31]: Just a reminder to update this before merging

File: .github/workflows/plugin_test.yaml [hunk 2]
```diff
@@ -28,13 +28,13 @@ jobs:
...
             tests_to_run: tests/.
           - plugin: pynxtools-stm
             branch: main
             tests_to_run: tests/.
           - plugin: pynxtools-xps
-            branch: main
+            branch: update-configs-after-mpes-changes
```
Comment (domna) [inline .github/workflows/plugin_test.yaml:37]: This needs to be updated before merging as well

**Review by domna — APPROVED** [review]

**Comment by domna** [comment]
Hey @lukaspie, I was thinking whether we could allow multiple branches in the CI/CD. Then it should check whether they exist and if not proceed to the main/next branch. That way it's possible to merge this without merging the failing ci in the readers or in pynx.

**Comment by lukaspie** [comment]
> Hey @lukaspie, I was thinking whether we could allow multiple branches in the CI/CD. Then it should check whether they exist and if not proceed to the main/next branch. That way it's possible to merge this without merging the failing ci in the readers or in pynx.

Yeah, sounds good, I also don't like how we have to do it currently. I will try to cook something up.

File: .github/workflows/plugin_test.yaml (hunk 1 above)
Comment (lukaspie) [reply .github/workflows/plugin_test.yaml]: done

File: .github/workflows/plugin_test.yaml (hunk 2 above)
Comment (lukaspie) [reply .github/workflows/plugin_test.yaml]: done

**Pull request merged**